packages/core/src/js/sdk.tsx

```diff
@@ -219,9 +219,23 @@ export async function close(): Promise<void> {
 
 /**
  * Captures user feedback and sends it to Sentry.
+ * @deprecated Use `Sentry.captureFeedback` instead.
  */
 export function captureUserFeedback(feedback: UserFeedback): void {
-  getClient<ReactNativeClient>()?.captureUserFeedback(feedback);
+  const feedbackParams = {
+    name: feedback.name,
+    email: feedback.email,
+    message: feedback.comments,
+    associatedEventId: feedback.event_id,
+  };
+  captureFeedback(feedbackParams);
+}
+
+/**
+ * Captures user feedback and sends it to Sentry.
+ */
+export function captureFeedback(feedbackParams: SendFeedbackParams): void {
+  getClient<ReactNativeClient>()?.captureFeedback(feedbackParams);
 }
```

Comment:
Could we directly use `captureFeedback` from `@sentry/core`.

Reply:
Updated with da0d4ac091e20b0232f841ff9a5597597e47070c